responsive topbar/index.html

```diff
@@ -0,0 +1,362 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="topbar">
+        <input type="checkbox" id="check" />
+        <div class="logo"></div>
+        <div class="menu">
+          <a href="#">Item1</a>
+          <a href="#">Item2</a>
+          <a href="#">Item3</a>
+        </div>
```

Comment:
to wszystko powinno byc w `nav`. masz poczytaj sobie tutaj o tym https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav